test/fixtures/logs/tasks.js

```diff
@@ -0,0 +1,56 @@
+export const taskLogs = [
+  {
+    meta: {
+      userId: "aaL1MXrpmnUNfLkhgXRj",
+      taskId: "P8ZZX2ef8heTs0erA6a8",
+      username: "shubham-sharma",
+    },
+    type: "task",
```

Comment:
- is there a reason why we're having the type capitalized (REQUEST_CREATED) in `logs/requests.js` not here?
- is the `type` here an enum?

Reply:
this is how already data is getting stored in logs collection. I'm using the existing data only. 
Will be creating one task for this refactoring of logs collection alter on, this was even discussed in the doc phase.

Reply:
https://github.com/Real-Dev-Squad/website-backend/issues/1978

Raised the ticket. Once this task is done, will discuss this one.